### ISSUE TITLE
train_loss  and train accuracy is very good,but val_loss is hign

### ISSUE THREAD

**Issue by wzqy56789** [MaybeShewill-CV/CRNN_Tensorflow#388]
I create 384 train samples and 96 val sapmles and every sample have 10 chinenese characters。Total chinenese characters is 50，it is mean __C.ARCH.NUM_CLASSES =51.  And __C.ARCH.INPUT_SIZE = (220, 32)，__C.ARCH.SEQ_LENGTH = 55 。

THEN I  have trained 1000 EPOCHS，BATCH_SIZE=32.  The train_loss is around 0.4  and train accuracy  is around 0.99，but  val_loss  is above 50，val accuracy is aroud 0.02。and test is not good even using train jpg。

My question ：What should I do to improve val accuracy  and test accuracy  ？

**Comment by wzqy56789** [comment]
I modify test_shadownet.py follows, it can correctly test image and val image. Why ?

net = crnn_net.ShadowNet(
        #phase='test',
        phase='train',

**Comment by MaybeShewill-CV** [comment]
@wzqy56789 check if your test input image use the same preprocess function as training procedure:)

**Comment by wzqy56789** [comment]
I found it that  accuracy of test with batch_normalization is hign.

**Comment by MaybeShewill-CV** [comment]
@wzqy56789 during testing process bn params should be frozen. I did not meet such a problem when i train the model:)
